bigbluebutton-html5/imports/ui/components/join-handler/component.jsx

```diff
@@ -178,19 +177,38 @@ class JoinHandler extends Component {
 
     setLogoutURL(response);
 
+    const setdefaultScreens = (resp) => {
+      const { meetingID: meetingId, role } = resp;
+
+      return new Promise((resolve) => {
+        if (meetingId && role === 'MODERATOR') {
+          makeCall('setDefaultScreens', meetingId).then(r => resolve(r));
+        }
+
+        resolve(true);
+      });
+    };
+
     if (response.returncode !== 'FAILED') {
       await setAuth(response);
+      await setdefaultScreens(response);
 
       setBannerProps(response);
       setLogoURL(response);
       setModOnlyMessage(response);
       logUserInfo();
 
       Tracker.autorun(async (cd) => {
-        const user = Users.findOne({ userId: Auth.userID, approved: true }, { fields: { _id: 1 } });
+        const user = Users.findOne({
+          userId: Auth.userID,
+          approved: true,
+        }, {
+          fields: { _id: 1, presenter: 1 },
```

Comment:
we should remove `presenter: 1` as it was not there earlier

Reply:
removed